### PR TITLE
Update qwen2.md

### PR DESCRIPTION
**Pull request by ArtificialZeng**
outdated description

# What does this PR do?

Fixes # (issue)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [x] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Review by ArtificialZeng — COMMENTED** [review]
outdated usage

**Review by amyeroberts — COMMENTED** [review]
Hi @ArtificialZeng, thanks for opening a PR! 

The linked to checkpoint Qwen/Qwen2-7B-Chat doesn't appear to exist on the hub

**Comment by ArtificialZeng** [comment]
> Hi @ArtificialZeng, thanks for opening a PR!
> 
> The linked to checkpoint Qwen/Qwen2-7B-Chat doesn't appear to exist on the hub

it's just a typo.

**Comment by amyeroberts** [comment]
@ArtificialZeng, I'm sorry, I don't understand. What is "just a typo"?

**Review by ArtificialZeng — COMMENTED** [review]
that's good now

**Comment by ArtificialZeng** [comment]
> @ArtificialZeng, I'm sorry, I don't understand. What is "just a typo"?

I have amended it. pls see this again. thank you for your patience

File: docs/source/en/model_doc/qwen2.md [hunk 1]
```diff
@@ -35,8 +35,8 @@ In the following, we demonstrate how to use `Qwen2-7B-Chat-beta` for the inferen
 >>> from transformers import AutoModelForCausalLM, AutoTokenizer
 >>> device = "cuda" # the device to load the model onto
 
->>> model = AutoModelForCausalLM.from_pretrained("Qwen/Qwen1.5-7B-Chat", device_map="auto")
->>> tokenizer = AutoTokenizer.from_pretrained("Qwen/Qwen1.5-7B-Chat")
+>>> model = AutoModelForCausalLM.from_pretrained("Qwen/Qwen2-7B-Instruct", device_map="auto")
```
Comment (amyeroberts) [inline docs/source/en/model_doc/qwen2.md:38]: Thanks for updating - can you update the lines above introducing the code too?

**Review by amyeroberts — APPROVED** [review]
Thanks for updating! Just one last thing to do and then it's good to merge

File: docs/source/en/model_doc/qwen2.md (hunk 1 above)
Comment (ArtificialZeng) [reply docs/source/en/model_doc/qwen2.md:38]: Of Course, It's my honor to do so. Thank you sis.

**Review by ArtificialZeng — COMMENTED** [review]
good to go

**Pull request merged**